doc/source/getting_started/introduction.ipynb

```diff
@@ -0,0 +1,496 @@
+    "See more on [data structures in the User Guide](../docs/user_guide/data_structures.rst).\n",
+    "</div>\n",
+    "\n",
+    "\n",
+    "Let's see how this works in practice.\n",
+    "\n",
+    "## Reading and writing files\n",
+    "\n",
+    "First, we need to read some data.\n",
+    "\n",
+    "### Read files\n",
+    "\n",
+    "Assuming we have a file containing both data and geometry (e.g. GeoPackage, GeoJSON, Shapefile), we can easily read it using `geopandas.read_file` function, which automatically detects filetype and creates a `GeoDataFrame`. In this example, we'll use `\"nybb\"` dataset, a map of New York boroughs which is part of GeoPandas installation. Therefore we need to get the path to the actual file. With your file, you specify a path as a string (`\"my_data/my_file.geojson\"`)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import geopandas\n",
+    "\n",
+    "path_to_data = geopandas.datasets.get_path(\"nybb\")\n",
+    "gdf = geopandas.read_file(path_to_data)\n",
+    "\n",
+    "gdf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Write files\n",
+    "\n",
+    "Writing a `GeoDataFrame` back to file is similarly simple, using `GeoDataFrame.to_file`. The default file format is Shapefile, but you can specify your own using `driver` keyword."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gdf.to_file(\"my_file.geojson\", driver=\"GeoJSON\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "User Guide\n",
+    "    \n",
+    "See more on [reading and writing data in the User Guide](../docs/user_guide/io.rst).\n",
+    "</div>\n",
+    "\n",
+    "\n",
+    "\n",
+    "## Simple methods\n",
+    "\n",
+    "Now we have our `GeoDataFrame` and can start working with its geometry. \n",
+    "\n",
+    "Since we have only one geometry column read from the file, it is automatically seen as the active geometry and methods used on `GeoDataFrame` will be applied to the `\"geometry\"` column.\n",
+    "\n",
+    "### Measuring area\n",
+    "\n",
+    "To measure the area of each polygon (or MultiPolygon in this specific case), we can use `GeoDataFrame.area` attribute, which returns a `pandas.Series`. Note that `GeoDataFrame.area` is just `GeoSeries.area` applied to an active geometry column.\n",
+    "\n",
+    "But first, we set the names of boroughs as an index, to make the results easier to read."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gdf = gdf.set_index(\"BoroName\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gdf[\"area\"] = gdf.area\n",
```

Comment:
Question here could be if we generally want to use `gdf.area` vs `gdf.geometry.area`. 

The first is easier/shorter, the second is more explicit (and can avoid potential confusion if you have a column called area)

Reply:
I would keep `gdf.area`. I tried to explain the concept of active geometry so I hope it is clear what is going on.